bindings.sh

```diff
@@ -13,4 +17,4 @@ done < .env.local
 
 bindings=$(echo $bindings | sed 's/[[:space:]]*$//')
 
-echo $bindings
+echo "$bindings --binding SUPABASE_URL=$SUPABASE_URL --binding SUPABASE_ANON_KEY=$SUPABASE_ANON_KEY"
```

Comment:
The above loop already adds these variables. Why add them again?

Reply:
That's fair. Will double check